.circleci/config.yml

```diff
@@ -17,19 +29,12 @@ workflows:
       - test-lint:
           requires:
             - prep-deps-npm
-      # - test-deps:
-      #     requires:
-      #       - prep-deps-npm
       - test-e2e-chrome:
           requires:
             - prep-deps-npm
       - test-e2e-firefox:
           requires:
             - prep-deps-npm
-      # - test-e2e-beta-drizzle:
```

Comment:
why are we removing the drizzle tests as part of this PR?

Reply:
That's a good catch, I'll add these back